content/en/cloud_cost_management/google_cloud.md

```diff
@@ -152,3 +156,4 @@ You can visualize your ingested data using the following cost types:
 [12]: /integrations/google_cloud_platform/
 [13]: /cloud_cost_management/google_cloud/#provide-service-account-necessary-permissions
 [14]: /cloud_cost_management/google_cloud/#enable-detailed-usage-cost-export
+[15]: https://docs.datadoghq.com/cloud_cost_management/container_cost_allocation/#tags
```

Comment:
Is this intentionally referencing the #tags section or did you mean to reference the the overall docs page?
```suggestion
[15]: https://docs.datadoghq.com/cloud_cost_management/container_cost_allocation/
```